docs/bn/week06/06-1.md

```diff
@@ -0,0 +1,186 @@
+##নেগেটিভ মাইনিং
+শেষ অনুচ্ছেদে আমরা আলোচনা করেছি যে কিভাবে টেস্টিং পিরিয়ডে মডেল অধিক সংখ্যক ফলস পজিটিভস নিরীক্ষার মুখোমুখি হতে পারে, যেহেতু নন-ফেস অবজেক্টের বিভিন্ন প্রকারে মুখাবয়বের সাথে সাদৃশ্য থাকতে পারে। কোন ট্রেইনিং সেট সব রকম সম্ভাবনার নন-ফেস অবজেক্ট অন্তর্ভুক্ত করবেনা যেগুলো দেখতে চেহারার মত। আমরা এই সমস্যা নিরসনে নেগেটিভ মাইনিং ব্যবহার করতে পারি। নেগেটিভ মাইনিং প্রক্রিয়াতে, আমরা নন-ফেস ক্ষুদ্র অংশ গুলোর দ্বারা একটি নেগেটিভ ডেটাসেট প্রস্তুত করি, যেগুলো মডেল ভুলক্রমে মুখাবয়ব হিসেবে চিহ্নিত করেছে। মডেলটিকে এমন সব জ্ঞাত ইনপুটের উপর চালানো হয়েছে যেগুলোতে জ্ঞাতসারে নন-ফেস অংশ আছে এবং ডেটা সংগ্রহে এই পদ্ধতি গ্রহণ করা হয়েছে। এরপর ডিটেক্টরটি পুনরায় ট্রেইন করা হয়েছে নেগেটিভ ডেটাসেট ব্যবহার করে। আমরা এই পদ্ধতি পুনরাবৃত্তি করতে পারি, যাতে করে ফলস পজিটিভের বিপরীতে মডেলের সঠিক নির্ধারণ ক্ষমতা বৃদ্ধি পায়। 
+
+
+## Semantic segmentation
+
+<!--Semantic segmentation is the task of assigning a category to every pixel in an input image.-->
+ সেমান্টিক সেগমেন্টেশন একটি ইনপুট  চিত্রের প্রতিটি পিক্সেলে ভিন্ন ভিন্ন শ্রেণী নির্দিষ্ট করে দেয় । 
+
+### [CNN for Long Range Adaptive Robot Vision](https://www.youtube.com/watch?v=ycbMGyCPzvE&t=1669s)
+দীর্ঘ পরিসর ব্যাপী অভিযোজিত রোবট ভিশন
+
+<!--In this project, the goal was to label regions from input images so that a robot can distinguish between roads and obstacles. In the figure, the green regions are areas the robot can drive on and the red regions are obstacles like tall grass. To train the network for this task, we took a patch from the image and manually label it traversable or not (green or red). We then train the convolutional network on the patches by asking it to predict the color of the patch. Once the system is sufficiently trained, it is applied to the entire image, labeling all the regions of the image as green or red.-->
+এই প্রজেক্টের উদ্দেশ্য ছিল ইনপুট চিত্রের থেকে বিশেষ অংশ গুলো লেবেল করে দেয়া, যাতে করে একটি রোবট রাস্তা এবং প্রতিবন্ধক আলাদা করে চিনতে পারে। নিচের চিত্রে, সবুজ অংশ গুলোতে রোবটটি গাড়ি চালাতে পারবে, এবং লাল অংশের মধ্যে আছে লম্বা ঘাসের ন্যায় প্রতিবন্ধক। এই করণীয় কাজের উদ্দেশ্যে নেটওয়ার্ক ট্রেইন করার জন্য আমরা ইনপুট চিত্রের একটি অংশকে  গাড়ি চালনার উপযুক্ত কিনা (সবুজ কিংবা লাল) হাতে করে লেবেল করে দিয়েছি। এরপর আমরা কনভলিউশনাল নেটওয়ার্ক লেবেল করা চিত্রের ওপর ট্রেইন করি এবং নেটওয়ার্কটিকে চিত্রের লেবেল করা অংশের রঙ অনুমান করতে বলি। 
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-1/5mM7dTT.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 4:</b> CNN for Long Range Adaptive Robot Vision (DARPA LAGR program 2005-2008)
```

Comment:
All captions need a translation.

Reply:
Checked all captions and added translation. Let me know if there's anything more left.